src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -138,6 +130,22 @@ AutomatedRssDownloader::~AutomatedRssDownloader()
     delete m_episodeRegex;
 }
 
+void AutomatedRssDownloader::connectRuleFeedSlots()
+{
+    qDebug() << Q_FUNC_INFO << "Connecting rule and feed slots";
+    connect(ui->listRules, SIGNAL(itemSelectionChanged()), this, SLOT(updateRuleDefinitionBox()), Qt::UniqueConnection);
+    connect(ui->listRules, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleRuleCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+    connect(ui->listFeeds, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleFeedCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+}
```

Comment:
Ok. I was looking at the Qt5 docs which describe it a bit different so ok. Leave it as-is. Same for the `this` pointer.